README.md

```diff
@@ -30,8 +33,8 @@ To edit this project:
 - Github: [@HeDevedUp](https://github.com/HeDevedUp)
 - Linkedin: [Jesse Uzoma](https://www.linkedin.com/in/jesseuzoma/)
 
-## :star2: Show your support
-Give a :star: if you like my project!
+## :star2: Show your support:
+- Give a :star: if you like my project!
 
-## :pencil: License
-This project is [MIT](https://github.com/Gopxfs/fire-data-app/blob/main/LICENSE) licensed.
+## :Acknowledgments:
+- Credit goes to Nelson Sakwa for the design inspiration.
```

Comment:
Well done giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) 🤓